### DISCUSSION TITLE
Problemas con la lectura driver

### DISCUSSION THREAD

**Discussion by jainerms** [rramosp/ai4eng.v1#80] [04 - GESTIÓN DE DATOS]
cordial saludo
El siguiente foro  es para solicitar apoyo con la lectura del driver para la lectura de los archivos, no me esta dejando, cargo el driver y no me da, me sale ese error,
haber si me podria ilustrar como solucionar este detallito, solo es que me logree leer los archivos, para porder ir mondeando el codigo

![image](https://user-images.githubusercontent.com/95552158/145246544-b097629d-408a-4e90-a9bf-34ca13438167.png)

Ya he mirado varias opciones y estoy quedado en esa parte

Quedo atento a cualquier novedad, muchisimas gracias

**Comment by Felipe-RA** [comment]
Saludos jainerms

Mi método preferido para incorporar archivos desde Google Drive en Google Colab es *montando* tu Drive.

En este notebook oficial puedes encontrar un tutorial de cómo hacerlo:

https://colab.research.google.com/notebooks/io.ipynb


Una vez que hayas montado tu Drive podrás acceder su contenido usando la ruta hacia el fichero que necesites.

Recuerda que Colab tiene un explorador de archivos, en caso de que lo llegues a necesitar.
